blocks/horizontal-list/horizontal-list.css

```diff
@@ -0,0 +1,64 @@
+/* Horizontal List */
+.horizontal-list ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.horizontal-list.careers-tags ul {
+    font-size: var(--body-font-size-xs);
+}
+
+.horizontal-list li {
+    display: inline-block;
+    margin-right: 1rem;
+    text-align: center;
+}
+
+.horizontal-list li::before {
+    display: inline-block;
+    content: '';
+    background: inherit;
+    background-color: var(--text-color);
+    height: 12px;
+    width: 12px;
+    margin-right: 4px;
+    border-radius: 50%;
+}
+
+.horizontal-list.careers-tags li.blue {
```

Comment:
`careers` is a content specific term. I'd suggest to prefer generic class names since the block can be used beyond just `careers` content.

Reply:
This was used before I swithed to the network-item block, tbh I forgot to delete it, but I can keep the horizontal-list block if it seems useful and remove any mentions of the `careers` page